### ISSUE TITLE
[PCB] fix missing connection on attiny devmode level shifter

### ISSUE THREAD

**Issue by graemeDnf** [GlenNicholls/solar_tracking_project#33]
![1ea9cd10-919c-40cf-8ffb-1414e7083deb](https://user-images.githubusercontent.com/17101356/47735986-4abad080-dc33-11e8-9a9e-a77391600c9b.jpg)
Connection was deleted between low side of level shifter and 3.3v rail

**Comment by graemeDnf** [comment]
PCB file updated and pushed, awaiting confirmation of bluewire fix on existing board

**Comment by GlenNicholls** [comment]
yeah that's gonna be a bit if you want to CALM DOWN

**Comment by GlenNicholls** [comment]
Just to confirm, the highlighted via connects to the left pad of the resistor right above it?

**Comment by graemeDnf** [comment]
Correct. This is the bottom copper though, so the directions will be flipped when you look at the bottom side of the actual board. Also, it will probably be easiest to solder that resistor pad to the left pad of one of the neighboring resistors (same net) rather than the via
